src/components/Tree/styled/inputContainer.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const InputContainer = styled.div`
+    display: flex;
+    position: relative;
+    align-items: center;
+    justify-content: center;
+    width: 20px;
+    margin-right: 8px;
```

Comment:
I think there is no need to use a wrapper, instead, you can make the PrimitiveCheckbox a styled and pass your own styles to it since it already has a wrapper which receives a style prop.
Also, the only style you need to pass is the `margin-right: 8px;`